metadata-service/graphql-servlet-impl/src/main/java/com/datahub/graphql/GraphQLController.java

```diff
@@ -126,9 +126,20 @@ void getGraphQL(HttpServletRequest request, HttpServletResponse response) {
     throw new UnsupportedOperationException("GraphQL gets not supported.");
   }
 
+  private void observeErrors(ExecutionResult executionResult) {
+    executionResult.getErrors().forEach(graphQLError -> {
+      String errorType = graphQLError.getErrorType().toString();
```

Comment:
So we also have our own exception type called DataHubGraphQLError that has an "error code" field. This would be useful to extract as it can show error codes like 401, 404, 500 etc. 

If the first error is _not_ an instance of DataHubGraphQLError, then we can increment he "500" error code bucket

Reply:
I'm seeing metrics like this if I run it locally:
`metrics_com_datahub_graphql_GraphQLController_errorType_DataFetchingException_Count`
`DataFetchingException` is the result of `DataHubGraphQLError.getErrorType()`: [here](https://github.com/datahub-project/datahub/blob/55357783f330950408e4624b3f1421594c98e3bc/datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/exception/DataHubGraphQLError.java#L63)